debian/autobake-deb.sh

```diff
@@ -122,17 +132,21 @@ in
     ;;
   # Ubuntu
   "bionic")
+    switch_back_to_ncurses5
     remove_rocksdb_tools
     [ "$architecture" != amd64 ] && disable_pmem
     ;&
   "focal")
+    switch_back_to_ncurses5
```

Comment:
these fall though too.

Why is Ubuntu being forced back to ncurses5 when it has ncurses (6.x+)? We'll need to manually add them to bb however (like https://github.com/MariaDB/buildbot/pull/168).

Reply:
Ain't they are fetch on build time? I'll  make Pull for BB also.

Reply:
It seems only Buster (which I'll make remove PR soon) should have Ncurses 5 others have Ncurses 6.2